cinema/views.py

```diff
@@ -1,11 +1,13 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets
+from rest_framework import viewsets, mixins
+from rest_framework.authentication import TokenAuthentication
 from rest_framework.pagination import PageNumberPagination
+from rest_framework.permissions import IsAuthenticated
```

Comment:
The import of `IsAuthenticated` is not necessary for the `OrderViewSet` since it already uses `IsAdminOrIfAuthenticatedReadOnly` from `cinema.permissions`. Consider removing this import if it's not used elsewhere in the file.